Dockerfile

```
@@ -12,10 +12,17 @@ LABEL "com.github.actions.color"="yellow"
 
 LABEL "maintainer"="Scott Brenner <scott@scottbrenner.me>"
 
-RUN apk --no-cache add nodejs npm python3 py3-pip
+RUN apk --no-cache add nodejs npm python3 py3-pip git make musl-dev go bash
```

```suggestion
RUN apk --no-cache add nodejs npm python3 py3-pip git make musl-dev go
```
Is bash necessary here?